openstudiocore/src/model/test/ElectricEquipmentITEAirCooled_GTest.cpp

```diff
@@ -264,5 +264,22 @@ TEST_F(ModelFixture, ElectricEquipmentITEAirCooled_inletportlist) {
 
 }
 
-// life cycle cost
+// test ITE approach temperature schedule
+TEST_F(ModelFixture, ElectricEquipmentITEAirCooledDefinition_approachtemperatureschedule) {
+  Model model;
+
+  ElectricEquipmentITEAirCooledDefinition definition(model);
+  ElectricEquipmentITEAirCooled electricEquipmentITEAirCooled(definition);
+  ScheduleCompact supplydeltasch(model, 5);
+  ScheduleCompact returndeltasch(model, -5);
```

Comment:
(Side note: prefer using actual `double` (that is `5.` or `5.0`) to avoid unecessary int to double implicit conversions)